ansibleinviewer/tests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+parameterized==0.7.0
+pytest==5.2.2
```

Comment:
In case of this module I would make the requirements as loose as possible as the module will be probably installed together with other python modules with their requirements. Forcing concrete versions would force the user to use a separate venv just for this module.

Reply:
I added requirements up to second version tag.